tests/test_outdated.py

```diff
@@ -24,16 +24,13 @@ class Package:  # noqa: D101
 
 
 allowed_outdated: set[str] = {
-    "python",  # ignore 3.12
     "conda",
     "mamba",  # conda/mamba conflict with tensorflow and VTK
```

Comment:
Do you think we could resolve this somehow?

Also I started transitioning away from `mamba` again in my personal dev environments, since `conda` now uses the `libmamba` solver by default

… this just to suggest that we may consider simply ditching `mamba` if it causes trouble!